puppet/zulip/files/cron.d/archive-messages

```diff
@@ -0,0 +1,6 @@
+SHELL=/bin/bash
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+USER=zulip
+
+# Daily cron job for retention policy.
+0 6 * * * zulip /home/zulip/deployments/current/manage.py archive_messages >/dev/null
```

Comment:
Do we need the `>/dev/null` here?  I suppose it's probably a good idea even if don't log anything; tracebacks would go to stderr anyway, right?

Reply:
Yeah, it shouldn't be outputting be anything to stdout anyway. Should we get rid of that? But yes, tracebacks shouldbe unaffected and go to stderr.

Reply:
I think it's probably better to have it, in case we add some basic informative logging.